### DISCUSSION TITLE
Supported way to block the adding of new buckets

### DISCUSSION THREAD

**Discussion by MANICX100** [ScoopInstaller/Scoop#5246] [Help and Questions]
Is there a supported way to disallow the adding of new buckets but still allow installations, updates and removals of existing apps?

**Comment by niheaven** [comment]
Disallow adding new buckets? What is your purpose here.

**Reply by MANICX100** [reply]
Infosec control over available applications for business use. I consider official buckets to be low risk and could envision a pre-approved set of additional tools in a custom bucket BUT wouldn't want people to be able to roll their own repos and/or get curious with non official buckets and potentially install an unwanted albeit non priveleged app. Naturally they run them anyway downloading it manually from the net, but lets not go thermonuclear paranoid and block everything.

Not possible?

**Comment by HUMORCE** [comment]
impossible, according to Scoop's design.
